package.json

```diff
@@ -52,6 +55,7 @@
     "eslint-config-terra": "^2.0.0",
     "rimraf": "^2.6.1",
     "shelljs": "^0.8.2",
-    "stylelint": "^9.3.0"
+    "stylelint": "^9.3.0",
+    "stylelint-test-rule-tape": "^0.2.0"
```

Comment:
I think this should be a dev-dependency?

Reply:
This is listed as a dev-dependency, github is just being a little deceiving with the line coverage preview

Reply:
ahhh. I see it now! I saw `find-up` and `postcss-value-parser` listed as dependencies and totally skipped the accordion line